pkg/topology/arangodb/arangodb-peer-change.go

```diff
@@ -0,0 +1,20 @@
+package arangodb
+
+func (m *arangoDB) peerChangeHandler(obj *message.PeerStateChange) {
```

Comment:
other methods use (a *arangoDB) instead of (m *arangoDB) it is a good practice to use the same, otherwise go linter will complain.